srcpkgs/python3-spake2/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-spake2'
+pkgname=python3-spake2
+version=0.8
+revision=1
+archs=noarch
+wrksrc="python-spake2-${version}"
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="python3-hkdf"
+short_desc="Pure-python implementation of the SPAKE2 Password-Authenticated Key Exchange algorithm"
```

Comment:
Too long. I'd suggest "SPAKE2 Password-Authenticated Key Exchange".
The prefix `python3-` and the `noarch` inherently tells it's pure python.